src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,171 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public MyLinkedList() {
+        head = null;
```

Comment:
if you want you can remove initialization from the constructor because `int` value by default has 0 value and object is null.

Reply:
done